source/_includes/installation/operating_system.md

```diff
@@ -73,50 +73,51 @@ To boot Home Assistant OS, the BIOS needs to have UEFI boot mode enabled and Sec
 1. Disable Secure Boot.
 ![Disable Secure Boot mode](/images/installation/intel-nuc-disable-secure-boot.jpg)
 
-1. Save the changes and exit.
+1. Save your changes and exit.
 
-- The BIOS configuration is complete.
+The BIOS configuration is now complete.
 
-As a next step, we need to write the Home Assistant Operating System image to the target boot medium. The HAOS has no integrated installer. This means the Operating System is not copied automatically to the internal disk.
+Next we need to write the Home Assistant Operating System image to the "boot medium", which is the medium your x86-64 hardware will boot from when it is running Home Assistant.
 
-- The "boot medium" is the medium your x86-64 hardware will boot from when it is running Home Assistant.
-- Typically, an internal medium is used for the x86-64 hardware. Examples of internal media are S-ATA hard disk, S-ATA SSD, M.2 SSD, or a non-removable eMMC.
-- Alternatively, an external medium can be used to boot HAOS such as a USB SDD (not recommended).
+<div class='note'>
+HAOS has no integrated installer that writes the image automatically, you must write it manually using e.g. Etcher.
```

Comment:
```suggestion
HAOS has no integrated installer that writes the image automatically, you must write it manually using e.g. Balena Etcher.
```

Reply:
Although that is a correct term, it is also referred to as `Etcher` on their own pages, and in most docs in this repo.

Reply:
You are right. I mainly saw balenaEtcher, but I only looked at one of their articles.